packages/openapi-v3/src/decorators/request-body.decorator.ts

```diff
@@ -157,4 +157,64 @@ export namespace requestBody {
       },
     });
   };
+
+  /**
+   * Define a requestBody of `file` type. This is used to support
+   * multipart/form-data based file upload.
+   *
+   * @example
+   * import {Request} from '@loopback/rest';
+   *
+   * ```ts
+   * class MyController {
+   *   @post('/pictures')
+   *   upload(
+   *     @requestBody.file()
+   *     request: Request,
+   *   ) {
+   *     // ...
+   *   }
+   * }
+   * ```
+   *
+   * @param properties - Optional description and required flag
+   */
+  export const file = (properties?: {
+    description?: string;
+    required?: boolean;
+  }) => {
+    return requestBody({
+      description: 'Request body for multipart/form-data based file upload',
+      required: true,
+      content: {
+        // Media type for file upload
+        'multipart/form-data': {
+          // Skip body parsing
+          'x-parser': 'stream',
+          schema: {
+            type: 'object',
+            properties: {
+              file: {
```

Comment:
nitpick: according to https://swagger.io/docs/specification/describing-request-body/file-upload/, the schema could be very different, the schema here is good enough as a default one, but I think it would be good to allow people provide their custom schemas(they are also responsible to make sure those schemas are valid)

Reply:
Sure. The `@requestBody.file` sugar decorator can take `multipart/form-data` as the default. For other forms, `@requestBody` continues to work.

Reply:
I added a comment in tsdocs.